api/questions/question.js

```diff
@@ -127,6 +130,27 @@ module.exports = (app) => {
             });
     });
 
+    route.get('/:case_id/questions', (req, res, next) => {
```

Comment:
has this been moved from somewhere else?

Reply:
Nope. We didn't have an end point for getting questions. We are using the `getQuestionsByCaseId` function to add them to the case file.